sklearn/utils/_estimator_html_repr.py

```diff
@@ -313,18 +319,14 @@ def _write_estimator_html(
   box-sizing: border-box;
   padding-bottom: 0.4em;
   background-color: white;
-  position: relative;
 }
 #$id div.sk-label label {
   font-family: monospace;
   font-weight: bold;
-  background-color: white;
```

Comment:
On some browsers this background color hides the dashed line on the top. Removing it is enough to fix it.